src/config/dynamic-theme-fixes.config

```diff
@@ -531,6 +531,15 @@ INVERT
 
 ================================
 
+arxiv.org
+
+CSS
+.abstract {
+     background-color: transparent !important;
```

Comment:
Why is it being set to `transparent`. It seem good for me without this fix.

Reply:
Without the fix, the abstract's background is white (https://static.arxiv.org/static/base/0.16.9rc3/css/abs.css)
```
#abs blockquote.abstract {
	font-size: 1.05em;
	margin-block-end: 21.6px;
	margin-block-start: 14.4px;
	margin-bottom: 21.6px;
	margin-inline-end: 40px;
	margin-inline-start: 40px;
	background-color: white;
	border-left: 0px;
	padding: 0px;
}
```

Reply:
It should be darkend by dark reader to the right color.
![image](https://user-images.githubusercontent.com/25481501/98302549-e28aec00-1fbc-11eb-9b30-8a62fbb15a03.png)

Reply:
That's weird. What I see is:
![2020-11-05_14-20](https://user-images.githubusercontent.com/24703459/98302902-1c82d080-1f72-11eb-8fe0-3d782232711d.png)

Reply:
Any custom settings? Did you try enable/disabling Dark reader.